From 2ccf7650893532149393671fd9928ab3c221d690 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@hadrons.org>
Date: Fri, 16 Aug 2019 00:34:47 +0200
Subject: [PATCH 56/60] tests: Fix test suite failure due parallel runs
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

By using the parent directory for «libls» tests, instead of the current
directory where other tests are generating files.
---
 tests/libls.sh | 26 +++++++++++++-------------
 1 file changed, 13 insertions(+), 13 deletions(-)

diff --git a/tests/libls.sh b/tests/libls.sh
index daf626c0..1326345c 100755
--- a/tests/libls.sh
+++ b/tests/libls.sh
@@ -56,19 +56,19 @@ $LS -alTt >/dev/null 2>&1
 # Several runs with different switches are compared by
 # a simple count of printed lines.
 #
-REPLY_a1=`$LS -a1`
-REPLY_A1=`$LS -A1`
-
-REPLY_C=`$LS -C`
-REPLY_Cf=`$LS -Cf`
-REPLY_Cr=`$LS -Cr`
-REPLY_Ct=`$LS -Ct`
-REPLY_x=`$LS -x`
-REPLY_m=`$LS -m`
-
-REPLY_l=`$LS -l`
-REPLY_lT=`$LS -l`
-REPLY_n=`$LS -n`
+REPLY_a1=`$LS -a1 ..`
+REPLY_A1=`$LS -A1 ..`
+
+REPLY_C=`$LS -C ..`
+REPLY_Cf=`$LS -Cf ..`
+REPLY_Cr=`$LS -Cr ..`
+REPLY_Ct=`$LS -Ct ..`
+REPLY_x=`$LS -x ..`
+REPLY_m=`$LS -m ..`
+
+REPLY_l=`$LS -l ..`
+REPLY_lT=`$LS -l ..`
+REPLY_n=`$LS -n ..`
 
 # In an attempt to counteract lack of subsecond accuracy,
 # probe the parent directory where timing is known to be more
-- 
2.26.0.292.g33ef6b2f38

